commands/serve.js

```diff
@@ -20,6 +20,7 @@ module.exports = new Command('serve')
   .option('-o, --host <host>', 'the host on which to listen (default: localhost)', 'localhost')
   .option('--only <targets>', 'only serve specified targets')
   .option('--except <targets>', 'serve all except specified targets')
+  .option('--hosting-public <targets>', 'override public directory')
```

Comment:
Would also recommend changing description to: 'override hosting directory for serving'